hecuba_py/src/hfilter.py

```diff
@@ -0,0 +1,241 @@
+import regex
+import inspect
+import re
+from hecuba import config
+from IStorage import IStorage
+
+
+def func_to_str(func):
+    func_string = inspect.getsourcelines(func)[0][0]
+    start, end = func_string.find("lambda"), func_string.rfind(",")
+    func_string = func_string[start:end]
+    func_vars = func_string[7:func_string.find(':')].replace(" ", "").split(',')
+    clean_string = func_string[func_string.find(':') + 1:].replace("\\n", '').replace("'", '')
+    return func_vars, clean_string
+
+
+def substit_var(final_list, func_vars, dictv):
```

Comment:
Please write a line of documentation explaining what does the method, since it seems complex and can't be understood easily